src/includes/getting-started-config/ruby.mdx

```diff
@@ -1,7 +1,7 @@
 Initialize the Sentry SDK early in your application's setup:
 
 ```ruby
-Raven.configure do |config|
+Sentry.init do |config|
```

Comment:
The code sample here needs to include enabling performance monitoring. We assume that *new* customers are on a plan that includes transactions. The Enabling Tracing page (renamed, FYI, to Performance Monitoring > Getting Started) is provided for customers who have an existing error monitoring SDK and need to only _add_ the performance monitoring package.